install_files/ansible-base/securedrop-configure.yml

```diff
@@ -60,6 +60,7 @@
 
     - name: securedrop_app_gpg_fingerprint
       prompt: Full fingerprint for the SecureDrop Application GPG Key
+      default: ' '
```

Comment:
Had to do this as a quick-fix for #1855 that was discovered while doing review. This commit should close out that issue but if there is disagreement there, I'm fine with leaving that issue open and creating a more exhaustive PR to do a cleaner fix.